### PR TITLE
fix: fixable flaws in Glosssary E-O

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/glossary/first_paint/index.html [hunk 1]
```diff
@@ -7,12 +7,12 @@
   - Performance
   - Web Performance
 ---
-<p><span class="seoSummary"><strong>First Paint</strong>, part of the <a href="/en-US/docs/">Paint Timing API</a>, is the time between navigation and when the browser renders the first pixels to the screen, </span>rendering anything that is visually different from what was on the screen prior to navigation. It answers the question "Is it happening?"</p>
+<p><span class="seoSummary"><strong>First Paint</strong>, part of the <a href="/en-US/docs/Web/PerformancePaintTiming">Paint Timing API</a>, is the time between navigation and when the browser renders the first pixels to the screen, </span>rendering anything that is visually different from what was on the screen prior to navigation. It answers the question "Is it happening?"</p>
```
Comment (nschonni) [inline files/en-us/glossary/first_paint/index.html:10]: Manually adjusted target

File: files/en-us/glossary/first_paint/index.html [hunk 2]
```diff
@@ -7,12 +7,12 @@
...
 
 <h2 id="See_also">See also</h2>
 
 <ul>
  <li><a href="/en-US/docs/Glossary/first_meaningful_paint">First meaningful paint</a></li>
  <li><a href="/en-US/docs/Glossary/First_contentful_paint">First contentful paint</a></li>
- <li><a href="/en-US/docs/">Paint Timing API</a></li>
+ <li><a href="/en-US/docs/Web/PerformancePaintTiming">Paint Timing API</a></li>
```
Comment (nschonni) [inline files/en-us/glossary/first_paint/index.html:17]: Manually adjusted target

File: files/en-us/glossary/gonk/index.html [hunk 3]
```diff
@@ -18,4 +18,4 @@
 
 <h2 id="Learn_more">Learn more</h2>
 
-<p><a href="https://developer.mozilla.org/en-US/Firefox_OS/Platform/Gonk">Gonk page on the Firefox OS zone</a></p>
+<p><a href="/en-US/docs/Archive/B2G_OS/Platform/Gonk">Gonk page on the Firefox OS zone</a></p>
```
Comment (nschonni) [inline files/en-us/glossary/gonk/index.html:21]: Manually adjusted as it gave the regular link but I found it wasn't in this repo, but the archived one

**Review by chrisdavidmills — APPROVED** [review]
LGTM

**Pull request merged**